### ISSUE TITLE
How to make double responses works on android.

### ISSUE THREAD

**Issue by ariestwn** [dialogflow/dialogflow-java-client-v2#29]
@abhi007tyagi hi, thankyou for your code. i've implemented it on my application. 

But i wonder how to make double responses on android. it would be great if it can.

![Screenshot_20190328-130118 png](https://user-images.githubusercontent.com/49016182/55134159-94640480-515a-11e9-9c81-2d90c6bd4ab1.png)

_Originally posted by @ariestwn in https://github.com/dialogflow/dialogflow-java-client-v2/issues/25#issuecomment-477462279_

**Comment by abhi007tyagi** [comment]
Double responses may not be possible at the moment as Dialogflow only supports 1-1 conversation.

**Comment by ariestwn** [comment]
what i mean is when i create response like

```
function newAgent(agent){
 agent.add('reply1');
 agent.add('reply2');
}
```
on Line integrations it works, but it doesn't work when i try it on android or web demo

**Comment by abhi007tyagi** [comment]
Yes, it is not supported by Dialogflow for Android/Java. You can raise an enhancement/feature for this.
